driver-kotlin-coroutine/src/integration/kotlin/com/mongodb/kotlin/client/coroutine/syncadapter/SyncMongoCluster.kt

```diff
@@ -115,33 +115,37 @@ internal open class SyncMongoCluster(open val wrapped: MongoCluster) : JMongoClu
         SyncChangeStreamIterable(wrapped.watch(clientSession.unwrapped(), pipeline, resultClass))
 
     override fun bulkWrite(models: MutableList<out ClientNamespacedWriteModel>): ClientBulkWriteResult {
-        org.junit.jupiter.api.Assumptions.assumeTrue(java.lang.Boolean.parseBoolean(toString()), "BULK-TODO implement")
-        TODO("BULK-TODO implement")
+        org.junit.jupiter.api.Assumptions.assumeTrue(
+            java.lang.Boolean.parseBoolean(toString()), "BULK-TODO Kotlin implement")
```

Comment:
I have updated the naming to specifically ignore Kotlin tests in the UnifiedTestRunner to ensure that only these tests are skipped on assumption errors until the Kotlin API is implemented.